src/snowflake/connector/azure_util.py

```diff
@@ -10,7 +10,12 @@
 from logging import getLogger
 from typing import TYPE_CHECKING, Any, Dict
 
-from azure.core.exceptions import HttpResponseError, ResourceNotFoundError
+from azure.core.exceptions import (
+    HttpResponseError,
+    ResourceNotFoundError,
+    ServiceRequestError,
+    ServiceResponseError,
```

Comment:
Are these new exception modules available only newer azure package or the existing ones since the oldest supported package but were not handled before?

Reply:
If they are new, you may need to bump up the minimum supported Azure package version.